### PR TITLE
Update ITAdmin-readiness.md

### PR DESCRIPTION
**Pull request by jhwang-ms**
Added two new Interactive Guides to this list.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/ITAdmin-readiness.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ Use the training resources to educate yourself about Teams, teach others or run
...
 |-----------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------|
 | Identity, Access management & Threat Protection for Teams                   | Learn how to use Microsoft admin centers and PowerShell to configure access and identity policies and Enable Office 365 Advanced Threat Protection for Teams                                                            | [Teams security guide](https://aka.ms/teamsesecurityinteractiveguide)         |
+| Data Loss Prevention (DLP) Policies for Microsoft Teams                     | Learn how to add Microsoft Teams to an existing DLP policy, as well as create a new policy, whether bawsed on a template or fully customized to your organization's data protection needs.                        | [Teams DLP guide](https://aka.ms/teamsdlpguide)         |
```
Comment (JohanFreelancer9) [inline Teams/ITAdmin-readiness.md:72]:
```suggestion
| Data Loss Prevention (DLP) Policies for Microsoft Teams                     | Learn how to add Microsoft Teams to an existing DLP policy, as well as create a new policy, whether based on a template or fully customized to your organization's data protection needs.                        | [Teams DLP guide](https://aka.ms/teamsdlpguide)         |
```

File: Teams/ITAdmin-readiness.md [hunk 2]
```diff
@@ -79,6 +80,7 @@ Use the training resources to educate yourself about Teams, teach others or run
...
 | Configuring Microsoft Teams for Firstline Workers                           | Learn how to configure and deploy the right Microsoft Teams environment for your Firstline Workforce.                                                                                                                  | [Configuring Microsoft Teams for Firstline Workers guide](https://aka.ms/ConfigFLWInteractiveGuide)              |
+| Identity for the Frontline workforce: Emplower IT, Managers, and Frontline Workers                    | Learn how to integrate Microsoft Teams with existing HR systems, configure lightweight delegated identity management, and configure simplified sign-in and seamless transitions for those using shared devices.                                                                                                                  | [Frontline Worker identity guide](https://aka.ms/FLWidentityguide)              |
```
Comment (JohanFreelancer9) [inline Teams/ITAdmin-readiness.md:83]:
```suggestion
| Identity for the frontline workforce: Empower IT, managers, and frontline workers                    | Learn how to integrate Microsoft Teams with existing HR systems, configure lightweight delegated identity management, and configure simplified sign-in and seamless transitions for those using shared devices.                                                                                                                  | [Frontline worker identity guide](https://aka.ms/FLWidentityguide)              |
```

Comment (JohanFreelancer9) [reply Teams/ITAdmin-readiness.md:83]: The link also seems broken. Please check and confirm.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Teams/ITAdmin-readiness.md (hunk 2 above)
Comment (jhwang-ms) [reply Teams/ITAdmin-readiness.md:83]: Just checked with content owner and the correct link is https://aka.ms/FLWIdentityIG.

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Teams/ITAdmin-readiness.md (hunk 2 above)
Comment (JohanFreelancer9) [reply Teams/ITAdmin-readiness.md:83]: @jhwang-ms - Thank you. Could you please commit my suggested changes if you agree with them? Many thanks.

File: Teams/ITAdmin-readiness.md (hunk 1 above)
Comment (JohanFreelancer9) [reply Teams/ITAdmin-readiness.md:72]: @jhwang-ms - please also commit this suggestion (fix for a typo). Thanks.

**Pull request merged**